py3status/__init__.py

```diff
@@ -16,6 +16,16 @@
 
 
 def main():
+    # detect gevent option early because monkey patching should be done before
+    # everything else starts kicking
+    if '-g' in sys.argv or '--gevent' in sys.argv:
+        try:
+            from gevent import monkey
+            monkey.patch_all()
+        except ImportError:
+            print('Requested gevent, but it is not installed!')
+            sys.exit(4)
```

Comment:
This seems very harsh!

We should allow py3status to run at a minimum just to display a nice error to the user.  Personally I'd like use to run but without the gevent, but add a small warning 'module' that can be clicked to hide.